dt/uni01alpha/networker/kustomization.yaml

```diff
@@ -0,0 +1,135 @@
+---
+apiVersion: kustomize.config.k8s.io/v1alpha1
+kind: Component
+
+transformers:
+  - |-
+    apiVersion: builtin
+    kind: NamespaceTransformer
+    metadata:
+      name: _ignored_
+      namespace: openstack
+    setRoleBindingSubjects: none
+    unsetOnly: true
+    fieldSpecs:
+      - path: metadata/name
+        kind: Namespace
+        create: true
+
+resources:
+  - dataplane-ssh-secret.yaml
+  - openstackdataplanenodeset.yaml
+  - openstackdataplanedeployment.yaml
+
+patches:
+  - target:
+      kind: OpenStackDataPlaneDeployment
+      name: .*
+    patch: |-
+      - op: replace
+        path: /metadata/name
+        value: networker-deploy
+
+  - target:
+      kind: OpenStackDataPlaneDeployment
+      name: .*
+    patch: |-
+      - op: replace
+        path: /spec/nodeSets
+        value: networker-nodes
```

Comment:
value: [networker-nodes]

Reply:
Thanks @karelyatin 

Should be fixed in the next commit.